src/dataviews/histogram-dataview-model.js

```diff
@@ -141,9 +141,9 @@ module.exports = DataviewModelBase.extend({
 
   _onChangeBinds: function () {
     DataviewModelBase.prototype._onChangeBinds.call(this);
-    this.bind('change:histogram_sizes', function (mdl, isSizesApplied, d) {
-      if (isSizesApplied) {
-        this.trigger('histogram_sizes', this);
+    this.bind('change:autoStyle', function (mdl, isAutoStyle, d) {
```

Comment:
AFAIK we're no longer changing this... so is this really needed? Is anybody listening to the `autoStyle` event that is being triggered below 👇 ?